packages/dom/src/dom.js

```diff
@@ -395,6 +395,40 @@ export function documentHasSelection() {
 	return range && ! range.collapsed;
 }
 
+/**
+ * Check whether the contents of the element have been entirely selected.
+ * Returns true if there is no possibility of selection.
+ *
+ * @param {Element} element The element to check.
+ *
+ * @return {boolean} True if fully selected, false if not.
```

Comment:
Oh, I just caught this is still "fully selected"–if you wanted to change it to "entirely selected" as well that might be nice for consistency. 😅